integration-testing/rnode_testing/rnode.py

```diff
@@ -87,8 +87,11 @@ def deploy_contract(self, contract):
     def propose_contract(self):
         return self.exec_run(f'{rnode_binary} propose')
 
-    def show_blocks(self):
-        return self.exec_run(f'{rnode_binary} show-blocks')
+    def show_blocks(self, blocks_num):
+        return self.exec_run(f'{rnode_binary} show-blocks -d{blocks_num}')
+
+    def show_block(self, block_hash):
+        return self.exec_run(f'{rnode_binary} show-block {block_hash}')
```

Comment:
@odeac the `exec_run` would log the exit_code and the output

Reply:
ok